transport-native-epoll/src/main/java/io/netty/channel/epoll/TcpMd5Util.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+import io.netty.util.internal.ObjectUtil;
+import java.net.InetAddress;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Map.Entry;
+
+final class TcpMd5Util {
+
+    static Collection<InetAddress> setTcpMd5Sig(AbstractEpollChannel channel, Collection<InetAddress> current,
```

Comment:
Can we rename the method ? Prefix something with set and return a new Collection sounds strange to me. Maybe something like `newTcpMd5Sigs` ?